app.js

```diff
@@ -13,9 +13,13 @@ const dirname = path.resolve();
 
 const app = express();
 const swaggerDoc = swaggerYaml.load('./swagger.yaml');
+const tutorialWinston = require("./config/tutorial.winston");
```

Comment:
We don't need to define loggers based on modules.

There will be only one logger `./config/logger`, Which should export a logger instance by default.
So we can use it like this:

```
logger.info('Something....');
```

CC: @jaimin45 @ashishkava